assets/js/selecting.js

```diff
@@ -19,33 +19,116 @@ function sslothable(s, n) {
     }
   
   }
-  function redirect() {
-      var cfw = document.getElementById("mySelecti");
-      var mainver = document.getElementById("mySelect");
-      var smallver = document.getElementById("mySelec");
-      var nver = document.getElementById("mySel");
-      var region = document.getElementById("mySe");
-      document.getElementById("result_invalidVersion").style.display = "none";
-		  document.getElementById("result_methodUnavailable").style.display = "none";
-      if (mainver.value == 0) {
-        document.getElementById("result_invalidVersion").style.display = "block";
-      }
-      else if (mainver.value < 11 || smallver.value < 4) {
-          window.location.href = "installing-boot9strap-(soundhax)";
-      }
-      else if (nver.value < 36 && smallver.value < 16) {
-          window.location.href = "installing-boot9strap-(soundhax-sp)";
-      }
-      else if (sslothable(smallver.value, nver.value)) {
-          window.location.href = "installing-boot9strap-(ssloth-browser)";
-      }
-      else if (region.value === "T") {
-          window.location.href = "seedminer-(twn)";
-      }
-      else if (region.value === "C") {
-          document.getElementById("result_methodUnavailable").style.display = "block";
-      }
-      else {
-          window.location.href = "seedminer";
-      }
+  function sslothablekor(s, n) {
+    if  
+        (
+        (s == 4 && n == 33) ||
+        (s == 5 && n == 34) ||
+        (s == 6 && n == 35) ||
+        (s == 7 && n == 35) ||
+        (s == 8 && n == 35) ||
+        (s == 9 && n == 36) ||
+        (s == 10 && n == 37) ||
+        (s == 12 && n == 38) ||
+        (s == 13 && n == 39)
+        ) {
+            return true;
+    }
+    else {
+        return false;
+    }
   }
+function redirect() {
+    var major = document.getElementById("major");
+    var minor = document.getElementById("minor");
+    var nver = document.getElementById("nver");
+    var region = document.getElementById("region");
+    document.getElementById("result_invalidVersion").style.display = "none";
+	document.getElementById("result_methodUnavailable").style.display = "none";
```

Comment:
~~don't want to be picky... but TAB mixed with spaces!~~

Reply:
mb, wasnt visible on my side